src/ansys/units/unit.py

```diff
@@ -319,6 +318,27 @@ def _new_units(self, __value, op):
 
         return Unit(self._condense(new_units))
 
+    def compatible_units(self):
+        """
+        Get a set of units with the same dimensions.
```

Comment:
```suggestion
        Get all units with the same dimensions.
```
also, add the return type `compatible_units(self) -> set:`

Reply:
Added these changes.